modules/serverless-connect-channel-source-ODC.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+//  * serverless/event_workflows/serverless-channels.adoc
+
+[id="serverless-connect-channel-source-ODC_{context}"]
```

Comment:
For all images, file, and header IDs, I'd use the lowercase `odc` to follow consistency with other acronyms that have been lowercased in those areas.

Reply:
Did a git grep and replaced ODC -> odc in any instances I found